central/networkgraph/entity/datastore/datastore_bench_test.go

```diff
@@ -18,69 +18,65 @@ import (
 	"go.uber.org/mock/gomock"
 )
 
-func BenchmarkNetEntityCreates(b *testing.B) {
-	ctx := sac.WithGlobalAccessScopeChecker(context.Background(),
+var (
+	globalAccessCtx = sac.WithGlobalAccessScopeChecker(context.Background(),
 		sac.AllowFixedScopes(
 			sac.AccessModeScopeKeys(storage.Access_READ_ACCESS, storage.Access_READ_WRITE_ACCESS),
 			sac.ResourceScopeKeys(resources.NetworkGraph),
 		))
+)
 
+func BenchmarkNetEntityCreates(b *testing.B) {
 	mockCtrl := gomock.NewController(b)
 	defer mockCtrl.Finish()
 
-	entities, err := testutils.GenRandomExtSrcNetworkEntity(pkgNet.IPv4, 10000, "c1")
-	require.NoError(b, err)
+	// Need to recreate the DB to avoid failure due to key conflicts from the reruns.
+	db := pgtest.ForT(b)
+
+	store := postgres.New(db.DB)
 
-	b.Run("createNetworkEntities", func(b *testing.B) {
-		// Need to recreate the DB to avoid failure due to key conflicts from the reruns.
-		db := pgtest.ForT(b)
+	treeMgr := networktree.Singleton()
+	defer treeMgr.DeleteNetworkTree(globalAccessCtx, "c1")
 
-		store := postgres.New(db.DB)
+	ds := NewEntityDataStore(store, mocks.NewMockDataStore(mockCtrl), treeMgr, connection.ManagerSingleton())
 
-		treeMgr := networktree.Singleton()
-		defer treeMgr.DeleteNetworkTree(ctx, "c1")
+	entities, err := testutils.GenRandomExtSrcNetworkEntity(pkgNet.IPv4, b.N, "c1")
```

Comment:
I think that a ClusterID is usually an UUID. Probably not a big deal, but this might increase the storage size of entities and also the IDs in the flows.

Reply:
Fair point, I'll use one of the IDs from the test fixtures package